src/olympia/constants/activity.py

```diff
@@ -954,6 +954,14 @@ class CLEAR_ADMIN_REVIEW_THEME(_LOG):
     admin_event = True
 
 
+class ADDON_SLUG_CHANGED(_LOG):
+    id = 182
+    format = _('Addon {addon} slug changed from {old} to {new}.')
```

Comment:
Should we include the user here?

Reply:
We should. Use `{user_responsible}` which is automatically added from the `user` we record on the `ActivityLog` when formatting (and that `user` itself is automatically added from current thread, if you don't explicitly set one when creating an `ActivityLog`)